common/app/templates/inlineJS/blocking/shouldEnhance.scala.js

```diff
@@ -15,6 +15,17 @@
         }
     };
 
+    var personDemandsFeatures = function () {
+        var locationHash = window.location.hash;
+        if (locationHash === '#featureson' || locationHash === '#nocore' || locationHash === '#gu.prefs.force-core=off') return true;
+        try {
+            var preference = window.localStorage.getItem('gu.prefs.force-core') || 'unknown';
+            return /"value":"off"/.test(preference);
```

Comment:
Minor comment, but `JSON.parse` wouldn't be expensive and would make this a tiny bit more readable. Happy to leave as is, though :smile: